modules/cli/src/main/scala/scala/cli/commands/ScalaCommand.scala

```diff
@@ -103,10 +103,14 @@ abstract class ScalaCommand[T](implicit parser: Parser[T], help: Help[T])
       )))
       .withTerminalWidthOpt {
         if (Properties.isWin)
-          if (coursier.paths.Util.useJni()) {
-            val size = coursier.jniutils.WindowsAnsiTerminal.terminalSize()
-            Some(size.getWidth)
-          }
+          if (coursier.paths.Util.useJni())
+            Try(coursier.jniutils.WindowsAnsiTerminal.terminalSize()).toOption.map(
+              _.getWidth
+            ).orElse {
+              val fallback = 120
+              System.err.println(s"Could not get terminal width, falling back to $fallback")
```

Comment:
Let's only print that… if a Java option is defined say (just in case, to help debugging), as this may be quite annoying if it's printed for every `scala-cli` run.
```suggestion
              if (java.lang.Boolean.getBoolean("scala.cli.windows-terminal.verbose"))
                System.err.println(s"Could not get terminal width, falling back to $fallback")
```